api/docker-compose.yaml

```diff
@@ -1,5 +1,15 @@
 version: "3"
 services:
+  scripts:
```

Comment:
I was using the "api" container for everything. This makes more sense for one-off containers.